content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx

```diff
@@ -4,14 +4,7 @@ title: Shadow database
 
 <TopBlock>
 
-<Admonition type="warning">
-
-**MongoDB not supported** <br />
-Prisma Migrate does not currently support the [MongoDB connector](/concepts/database-connectors/mongodb).
-
-</Admonition>
-
-Prisma Migrate uses a second, _temporary_ database when you run development-focused commands such as:
+Some development-focused commands for relational databases of Prisma Migrate use a second, _temporary_ database:
```

Comment:
Scope to relational databases and also de-emphasize the importance of the shadow database.